README.md

```diff
@@ -1,78 +1,78 @@
-# Welcome, Knative
+# Welcome, Native
 
-Knative (pronounced kay-nay-tiv) extends Kubernetes to provide a set of middleware 
+Native (pronounced kay-nay-tiv) extends Kubernetes to provide a set of middleware 
```

Comment:
Need to update the pronunciation, too.

Reply:
Thank you for catching this.  Much like myself, you have a keen eye.  I have pushed up a fix.